src/saml2/cert.py

```diff
@@ -336,31 +340,13 @@ def verify(self, signing_cert_str, cert_str):
             cert_algorithm = cert.get_signature_algorithm()
             if six.PY3:
                 cert_algorithm = cert_algorithm.decode('ascii')
+                cert_str = cert_str.encode('ascii')
```

Comment:
what if the certificate contains umlauts e.g. "ä"? Are they encoded in a special format?